Sources/Extensions/Shared/ColorExtensions.swift

```diff
@@ -34,13 +34,13 @@ public extension Color {
         return Color(red: red, green: green, blue: blue)!
     }
 
-    // swiftlint:disable next large_tuple
     /// SwifterSwift: RGB components for a Color (between 0 and 255).
     ///
-    ///		UIColor.red.rgbComponents.red -> 255
-    ///		NSColor.green.rgbComponents.green -> 255
-    ///		UIColor.blue.rgbComponents.blue -> 255
+    ///        UIColor.red.rgbComponents.red -> 255
```

Comment:
Do all these need to be spaced so much? If you only use 4 spaces, instead of 8, will the quick look comment still appear correctly?

Reply:
I'm sorry, this was not intentional. I've used Xcode's "Move Line Up / Down" and it changed formatting of lines in-between.
I've reverted those changes, but it seems that those header docs have inconsistent formatting, there is <space><tab>, <tab><tab>, 4x <space> or 8x <space>.